apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -331,6 +344,8 @@ def render(element_html, data):
             "options": option_set,
             "counter_type": counter_type,
             "no_counters": no_counters,
+            "options_placement_right": options_placement_right,
+            "options_placement_bottom": options_placement_bottom,
```

Comment:
A lot of code could be simplified if you pass the enum value to the template, like `"options_placement": options_placement.value`. Then the template becomes:

```
class="pl-matching-container pl-matching-container-{{options_placement}}"
```

Reply:
Sure, let me update that